data/json/mutations/mutations.json

```diff
@@ -568,7 +670,7 @@
     "id": "TERRIFYING",
     "name": { "str": "Terrifying" },
     "points": 1,
-    "description": "There's something about you that creatures find frightening, and they are more likely to try to flee.",
+    "description": "There's something about you that humans and post-humans find frightening, and they are more likely to try to flee.",
```

Comment:
This affects the moral of all creatures:
```
if( u->has_trait( trait_TERRIFYING ) ) {
            effective_morale -= 10;
        }
```
(From src/monster.cpp)
So "creatures" is more appropriate.